google/ggrc-core
# GGRC-6771/GGRC-7115 Add multiselect CA type

This PR is `on hold` until the BE ticket completed:

 - [x] GGRC-7115

# Issue description

Currently the checkbox option is limited to a True/False type response. It should be allowed to select  multiple options.

# Steps to test the changes

 - Open Admin page -> select GCA for any object type
 - Create GCA with type 'Multiselect'
 - Open Create object modal w/ GCA multiselect
 - Fill out all mandatory fields and GCA
 - Open created object info pane

Expected behavior: GCA multiselect is populated w/ selected values

Note: Described scenario is also applicable for LCA.

# Solution description

Replace ```input``` checkbox w/ multiselect component.

# Sanity checklist




# PR Review checklist
